app/experimenter/nimbus-ui/src/components/PageEditAudience/index.tsx

```diff
@@ -76,7 +76,19 @@ const PageEditAudience: React.FunctionComponent<RouteComponentProps> = () => {
   }, []);
 
   return (
-    <AppLayoutWithExperiment title="Audience" testId="PageEditAudience">
+    <AppLayoutWithExperiment
+      title="Audience"
+      testId="PageEditAudience"
+      redirect={({ status }) => {
+        if (status.review) {
+          return "request-review";
+        }
+
+        if (status.released) {
+          return "design";
+        }
+      }}
+    >
```

Comment:
Since we're doing this in 4 spots, you could DRY it up with a helper function to check for these/return what ya need.